.circleci/docker/build.sh

```diff
@@ -250,6 +250,15 @@ case "$image" in
     VISION=yes
     ROCM_VERSION=5.1.1
     ;;
+  pytorch-linux-focal-py3.7-gcc7)
```

Comment:
fyi you'll need to add this to the docker build workflow as well

Reply:
This particular image is already in the docker build workflow.